src/App.tsx

```diff
@@ -65,10 +65,15 @@ function App() {
     }
     
     const allFilterOptions = async () => {
-      const response = await api.getAllFilterOptions();
+      const { options, updatedAt, maxDate, minDate } = await api.getAllFilterOptions();
       dispatch({
```

Comment:
would it be better to just consolidate this into a single dispatch?

Reply:
I like keeping the state updates distinct